ext/drcovlib/drcovlib.h

```diff
@@ -91,6 +91,11 @@ typedef struct _drcovlib_options_t {
      * overrides that default.
      */
     const char *logdir;
+    /**
+     * By default, log file names are prefixed with "drcov".  This option overrides
+     * that default.
+     */
+    const char *logprefix;
     /**
      * This is an experimental option for running natively (i.e., not under
      * DynamoRIO control) until the nth thread, where n is the value of this
```

Comment:
The feature and code itself look good to me.  I would just suggest putting a note about the new option in the changelog so users will notice it.  The changelog is here: https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L151.  Please just add a list item to the bottom of that list saying that we added this new option to drcov.

Reply:
Thanks! I have added a note to the changelog per your request :-)